packages/config/src/projects/layer2s/types/Layer2.ts

```diff
@@ -81,11 +81,16 @@ export interface Layer2LivenessDisplay {
   }
 }
 
+export interface Layer2FinalityDisplayWarning {
+  type: 'project' | 'l2beat'
```

Comment:
I would say it should use `Sentiment` type. `project` and `l2beat` are really abstracted type names. Using `neutral` Sentiment should be good there.

Reply:
I've changed the type of the warning fields to - already present - `WarningWithSentiment`.